src/pages/RFOX/hooks/useStakingBalanceOfQuery.ts

```diff
@@ -1,12 +1,12 @@
 import { type AssetId, fromAssetId } from '@shapeshiftoss/caip'
-import { skipToken } from '@tanstack/react-query'
+import { skipToken, useQuery } from '@tanstack/react-query'
+import type { ReadContractQueryKey } from '@wagmi/core/query'
```

Comment:
type syntax change as well as using the right place as mentioned in the [docs](https://wagmi.sh/core/api/actions/readContract#tanstack-query).
Effectively ensures type import only (paranoia against evaluating things at runtime and getting into trouble with versions mismatch)